behind-the-veil-siteroot/imports/ui/components/util.jsx

```diff
@@ -66,4 +67,41 @@ export function useUserInfo() {
 
     console.log("Current logged in user:" + JSON.stringify(userInfo))
     return userInfo;
+}
+
+/**
+ * Gets a list of objects containing all the suggestions for the search bar
+ *
+ * @param type - either services or artists (which category to get suggestions for)
+ * @param usersData - all relevant user data to get suggestions for
+ * @param servicesData - all relevant services data to get suggestions for
+ * @returns {*[]} an array of objects. each object has a main and a sub attribute, both used as suggestions.
+ */
+export function useSearchSuggestions(type, usersData, servicesData) {
```

Comment:
So this is just a regular js function and not a hook since it doesn't call any other hooks. It's best to avoid prefixing non-hook functions with "use" to avoid confusion. Sorry for nitpicking hook name-related stuff again.

Reply:
haha it used to use, before i decided that it's better to have data passed in, no worries I will change it